src/is.ts

```diff
@@ -47,7 +47,7 @@ export const isDate = (payload: any): payload is Date =>
   payload instanceof Date && !isNaN(payload.valueOf());
 
 export const isError = (payload: any): payload is Error =>
-  payload instanceof Error;
+  payload instanceof Error || payload?.prototype instanceof Error;
```

Comment:
Removing this change doesn't fail any test for me (locally). Could you elaborate on what this does?